src/app/frontend/logs/logstoolbar/logstoolbar.scss

```diff
@@ -14,6 +14,10 @@
 
 $logs-toolbar-color-black: #000;
 $logs-toolbar-color-white: #fff;
+$logs-toolbar-text-font-size: 17px;
+$logs-toolbar-title-font-weight: 500;
+$logs-toolbar-select-margin-bottom: 15px;
+$logs-toolbar-select-margin-left: auto;
```

Comment:
This (and line 19) does not need to be in a variable. It is used only once and is pretty standard way of margining.